src/common/startup_check.py

```diff
@@ -53,6 +56,10 @@ async def startup_checks() -> None:
     logger.info('Checking consensus nodes network...')
     await _check_consensus_nodes_network()
 
+    if settings.split_rewards:
```

Comment:
You must also check that the wallet attached to the operator is set as a claimer in RewardSplitter: https://github.com/stakewise/v3-core/blob/main/contracts/misc/RewardSplitter.sol#L33